src/test/java/core/basesyntax/service/impl/CsvFileWriterServiceTest.java

```diff
@@ -0,0 +1,71 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.FileWriterService;
+import java.io.File;
+import java.util.Map;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class CsvFileWriterServiceTest {
+    private static final String DEFAULT_PATH = "src/test/resources/fileWriter/";
```

Comment:
I think you should create new file for output of your writer(name of file "default" doesn't give any info, and why you need another package fileWriter?

Reply:
In my code, I always clean the package after the test to check if my method creates a new file.
But on the downside, I have the other method which read from a file ("test.csv")

If I didn't use the separate packages, I would have some problems, since I would just clean all files